attribute-masking-service/README.md

```diff
@@ -35,37 +39,37 @@ The service performs two functions:
 | user                    | maskedLeafResource       | exception         | user
 | leafResource            |                          | serverMetadata    | leafResource
 | rules                   |                          |                   | rules
-(fields marked with * are acquired from headers metadata)
 
-The service takes in an `AttributeMaskingRequest` and a request `Token`.
-This request is persisted in a store as an `AuthorisedRequestEntity`.
-The `LeafResource` attached to the request is then masked, removing any excessive or sensitive metadata which a client may be prohibited from accessing.
-The resulting masked `LeafResource` is then attached to an `AttributeMaskingResponse` and outputted from the service.
-
-### Future Enhancements
-The masking operation may in the future apply attribute-level `Rule`s using the `User` and `Context` for fine-grained control over client metadata access.
+*token is in the message header's metadata
 
+The service reads a message in from the Kafka "rule" topic as a [AttributeMaskingRequest](src/main/java/uk/gov/gchq/palisade/service/attributemask/model/AttributeMaskingRequest.java).
+This request is persisted in a store as an`AuthorisedRequestEntity`.
+The `LeafResource` attached to the request is then masked, removing any excessive or sensitive metadata which a client may be prohibited from accessing.
```

Comment:
```suggestion
Each message will either be removed from the stream, if the resource cannot be accessed, or redact the resource metadata.
```

Reply:
Appreciate the sentence was poorly written, but the suggestion is not quite right either.  I've taken another go at it, see what you think.